azurerm/internal/services/cosmos/common/schema.go

```diff
@@ -123,7 +123,7 @@ func CosmosDbIndexingPolicySchema() *schema.Schema {
 					ValidateFunc: validation.StringInSlice([]string{
 						string(documentdb.Consistent),
 						string(documentdb.None),
-					}, false),
+					}, true),
```

Comment:
Why are we making this case insensitive? the SDK values should match what we get from the API?

Reply:
In preview version, the value is "consistent" and "none", but in current version, the value is "Consistent" and "None". To resolve the breaking change, I set it to be case insensitive?

Reply:
instead of making this case insensitive can we just correct the casing to the SDK values on read?

also - could we open an issue on the SDK or somewhere to track this discrepancy?

Reply:
The case has been change when the service has released 2020-04-01 stable version. The old 2020-04-01-preview version has been deleted. 
Since preview version permits breaking changes, shall I not open the issue?